test/models/work_test.rb

```diff
@@ -0,0 +1,66 @@
+require "test_helper"
+require 'pry'
+
+describe Work do
+  let(:work) { Work.new(title: 'test') }
+
+  it "must be valid" do
+    value(work).must_be :valid?
+  end
+
+  describe 'validations' do
+    before do
+      @work = Work.new(
+        title: 'test title')
+    end
+
+    it 'is valid when title is present and unique' do
+
+      is_valid = @work.valid?
+      expect( is_valid ).must_equal true
+    end
+
+    it 'is invalid with a non-unique title' do
+      @work.title = Work.first.title
+
+      is_valid = @work.valid?
+
+      expect( is_valid ).must_equal false
+      expect( @work.errors.messages ).must_include :title
+    end
+
+    it 'is invalid with a missing title' do
+      @work.title = nil
+
+      is_valid = @work.valid?
+
+      expect( is_valid ).must_equal false
+
+      expect ( @work.errors.messages ).must_include :title
+    end
+  end
+
+  describe 'list_works_by_category' do
+
+    it 'correctly filters the list for a category' do
+      book_list = Work.list_works_by_category('book')
```

Comment:
There's a lot of interesting test cases for your custom `Work` methods missing here. For finding the media spotlight, I would wonder:

- What happens if there are no works?
- What happens if there are works but no votes?
- What happens if two works have the same number of votes?

Similarly for your category and top ten methods, I would ask:

- What if there are no works of that category?
- What if there are less than 10 works?
- What if there's a tie for last place, e.g. works 9, 10 and 11 all have 0 votes?